server/info.go

```diff
@@ -801,3 +689,70 @@ func (s *infoServer) LatestBlockHeight(ctx context.Context) (uint64, error) {
 func (s *infoServer) BlockCacheSize(ctx context.Context) (int64, error) {
 	return s.cacheClient.ListSize(ctx, cache.KeyBlocks)
 }
+
+func (s *infoServer) storeEvents(ctx context.Context, logs []types.Log) error {
+	for i, log := range logs {
+		decodedLog, err := s.decodeEvent(ctx, &log)
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>G601: Implicit memory aliasing in for loop. (gosec)